test.js

```diff
@@ -121,6 +121,20 @@ test('.stopAndPersist() - with new symbol and text', macro, spinner => {
 	spinner.stopAndPersist({symbol: '@', text: 'all done'});
 }, /@ all done/);
 
+test('spinner with default indent (0)', t => {
+	const spinner = new Ora('foo');
+	t.falsy(spinner.indent);
+});
+
+test('spinner with indent (7)', t => {
```

Comment:
I'm struggling a bit with testing the actual indented output. I was trying something along the lines of the code below, but with no luck.

```javascript
test('indented spinner', async t => {
    const stream = getPassThroughStream();
    const output = getStream(stream);

    const spinner = new Ora('foo');
    spinner.stream = stream;
    spinner.indent = 3;
    spinner.color = false;
    spinner.enabled = true;
    spinner.start();

    stream.end();

    t.regex(await output, /\s{3}${spinnerChar} foo/);
});
```

This is the ava output:

```
   145:                                                    
   146:   t.regex(await output, /\s{3}${spinnerChar} foo/);
   147: });                                                

  Value must match expression:

    "⠋ foo"

  Regular expression:

    /\s{3}${spinnerChar} foo/

  awaitoutput
  => "⠋ foo"

  Test.<anonymous> (test.js:146:4)
  step (test.js:26:191)
```

Can someone give me a hand or point me in the right direction?